seed/challenges/03-back-end-development-certification/mongodb.json

```diff
@@ -20,12 +20,12 @@
         "Install MongoDB with this command: <code>sudo apt-get install mongodb-org</code>",
         "Set up your mongod alias by following the directions at <a href='https://community.c9.io/t/setting-up-mongodb/1717' target='_blank'>https://community.c9.io/t/setting-up-mongodb/1717</a>.",
         "Now start the tutorial by running <code>learnyoumongo</code>.",
-        "Whenever you run a command that includes <code>mongod</code> on c9.io, use <code>./mongod</code> instead.",
+        "Whenever you run a command that includes <code>mongod</code> on c9.io, use <code>./mongod</code> instead. Together with the commands used in Step 11, this adds additional parameters to the <code>mongod</code> command to allow MongoDB to run in the C9 environment.",
         "Note that you can resize the c9.io's windows by dragging their borders.",
         "Make sure that you are always in your project's \"workspace\" directory. You can always navigate back to this directory by running this command: <code>cd ~/workspace</code>.",
         "You can view this Node School module's source code on GitHub at <a href='https://github.com/evanlucas/learnyoumongo'>https://github.com/evanlucas/learnyoumongo</a>.",
         "Complete \"Mongod\"",
-        "Complete \"Connect\"",
+        "Complete \"Connect\" <strong>Note:</strong> Use the command <code>mongod --port 27017</code> in place of <code>mongod --port 27017 --dbpath=./data</code> to avoid a duplicate parameter error. <code>--dbpath=./data</code> is already included in the <code>./mongod</code> command.",
```

Comment:
add a couple of breaks and take this into a new line

![image](https://cloud.githubusercontent.com/assets/1884376/16728083/02a5f920-4782-11e6-908c-1cc353189bb9.png)